.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MAIN]
+
+# Specify a score threshold under which the program will exit with error.
+fail-under=10
```

Comment:
Does this mean that small errors are OK? I don't think that's ok.

Reply:
No more like the opposite... it means anything below score 10 (out of 10) fails